app/config/FrontendAppConfig.scala

```diff
@@ -112,9 +111,6 @@ class FrontendAppConfig @Inject()(configuration: Configuration, servicesConfig:
 
   override lazy val viewSubmittedReturnUrl: String = getString(ConfigKeys.viewVatReturnsHost) + getString(ConfigKeys.submittedReturnsUrl)
 
-  // Business Tax Account
-  override lazy val btaHomeUrl: String = getString(ConfigKeys.btaHost) + getString(ConfigKeys.btaHomeUrl)
```

Comment:
Can you also remove the BTA config from `application.conf`, `ConfigKeys`, `BaseISpec` and from app-config-base